CHANGELOG.md

```diff
@@ -0,0 +1,224 @@
+# Changelog
+
+`react-stripe-elements` adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## v3.0.0 - 2019-04-17
+
+### New Features
+
+- added a [changelog](/CHANGELOG.md)
+- added support for `stripe.handleCardPayment` and `stripe.createPaymentMethod`.
+  These methods allow you to easily integrate Stripe's new Payment Intents API.
+  Like `createToken` and `createSource`, these new methods will automatically
+  find and use a corresponding Element when they are called.
+
+  ```js
+    stripe.createPaymentMethod(
+      paymentMethodType: string,
+      paymentMethodDetails: Object
+    ): Promise<{error?: Object, paymentIntent?: Object}>
+
+    stripe.handleCardPayment(
+      clientSecret: string,
+      paymentMethodDetails: Object
+    ): Promise<{error?: Object, paymentIntent?: Object}>
+  ```
+
+  For more information, please review the Stripe Docs:
+
+  - [`stripe.createPaymentMethod`](https://stripe.com/docs/stripe-js/reference#stripe-create-payment-method)
+  - [`stripe.handleCardPayment`](https://stripe.com/docs/stripe-js/reference#stripe-handle-card-payment)
+  - [Payment Intents API](https://stripe.com/docs/payments/payment-intents)
+  - [Payment Methods API](https://stripe.com/docs/payments/payment-methods)
+
+### Breaking Changes:
+
+- If you were already using `handleCardPayment` or `createPaymentMethod` with
+  `react-stripe-elements`, you should upgrade your integration. These methods
+  will now automatically find and use valid Elements.
+- Beta versions of Payment Intents with Stripe.js are not supported.
```

Comment:
> Beta versions of Payment Intents

could mean a variety of things. I think we want to clarify this means beta flags passed in to Stripe.js rather than earlier API versions.

Also, I think that while "Payment Intent API" is ok, if we're talking about the object itself, it has to be "PaymentIntent". We can change in either direction.